### ISSUE TITLE
How can I connect 2 mambo drone simultaneously with 1 computer?

### ISSUE THREAD

**Issue by jwlee2017** [amymcgovern/pyparrot#42]
In my laptop, I have 2 lan card and connect it to 2 wi-fi sources at same time. 
If only one drone is connected (another wi-fi is connected to normal internet), there is no problem running example code. But if 2 different mambo drones are connected to laptop, program said "connection failed: did you remember to connect your  machine to the drone's wifi network?" not only when I tried to connect 2 mambo simultaneously but also when I run example file and tried to connect to 1 mambo.

Can I connect to 2 mambo drones simultaneously by one or more python prosseses? I want to use wi-fi because I need to get pictures from each camera.

**Comment by amymcgovern** [comment]
I don't think this is possible because both drones have the same IP address.  It is hard-coded into the drone.
